docs/schema/1.0/schema.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Draft verifiable credential schema using JSON interchange format",
+  "type": "object",
+  "properties": {
+    "type": {
+      "type": "string"
+    },
+    "modelVersion": {
+      "type": "string",
+      "pattern": "^\\d+\\.\\d+$"
+    },
+    "id": {
+      "type": "string",
+      "pattern": "^did:\\w+:[1-9A-HJ-NP-Za-km-z]{16,};id=[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12};version=\\d+.\\d+$"
```

Comment:
if you'd prefer we can drop the regex on this for now

Reply:
or make it something like 
```
"^\\w+;id=[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12};version=\\d+.\\d+$"
```

Reply:
I'd prefer we update it to be like iglu: https://github.com/snowplow/iglu-central/blob/master/schemas/com.snowplowanalytics.self-desc/instance/jsonschema/1-0-0#L15

https://github.com/snowplow/iglu/wiki/Self-describing-JSONs

Reply:
At first glance this looks interesting, but will need some more time to digest (and let's discuss in https://github.com/w3c-ccg/vc-json-schemas/issues/17)
To move forward sooner I'd like to either drop the pattern or keep as is. Fine to keep this open until 17 is resolved too..

Reply:
Lets merge this, and address it after discussion on #17